src/tests/math/test_linspace.f90

```diff
@@ -0,0 +1,191 @@
+program test_linspace
+    use stdlib_error, only: check
+    use stdlib_kinds, only: sp, dp
+    use stdlib_math, only: linspace 
+
+    implicit none
+    logical :: warn = .false.
+
+    ! Testing linspace.
+    !
+    ! For single and double precision, check if the beginning and end values are properly calculated
+    ! and make sure that the size of the result is as expected.
+    !
+    !
+    open(unit=9, file="test_linspace_log.txt", status="unknown") ! Log the results of the functio
```

Comment:
It is correct. But is it not advisable to use `newunit=...` instead?

Reply:
Adjusted in [973bd3c](https://github.com/fortran-lang/stdlib/pull/420/commits/973bd3c34fb8c8b4872e9c78008ed61d2f63ff70)